projects/ExtraRepository.cmake

```diff
@@ -0,0 +1,33 @@
+foreach(extra_repository ${EXTRA_REPOSITORIES})
```

Comment:
@awhiemstra: I understand. But couldn't we keep this kind of hack? I could move it out of /projects/*..
Don't have a magic glass sphere here, but maybe we'll get into a situation where this can be very handy in the future :confused: